### PR TITLE
Implement query to fetch village and patient names in the facility

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/8768/implement-type-ahead-search-suggestions-for-overdue-search

### CODE REVIEW

**Comment by janhavisinghh** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4123** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4123" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4123?utm_source=stack-comment).

File: app/src/main/java/org/simple/clinic/patient/Patient.kt [hunk 1]
```diff
@@ -366,6 +366,30 @@ data class Patient(
...
+    
+        SELECT DISTINCT fullName 
+        FROM Patient 
+        WHERE fullName IS NOT NULL 
+        AND registeredFacilityId = :facilityUuid 
+        OR assignedFacilityId = :facilityUuid
+        )
+     ORDER BY colonyOrVillage ASC   
+    """)
+    abstract fun fetchColonyOrVillageAndPatientNames(facilityUuid: UUID): List<String>
```
Comment (msasikanth) [inline app/src/main/java/org/simple/clinic/patient/Patient.kt:392]: Not so sure about the function name, feel like we are doing too many things in a single function.

Comment (Honey14) [reply app/src/main/java/org/simple/clinic/patient/Patient.kt:392]: We're loading village and patient names both, do you think `loadVillageAndPatientNames` would be better? @msasikanth @janhavisinghh

Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/patient/Patient.kt:392]: Yeah, that works `loadPatientAndVillageNames`

**Review by Honey14 — DISMISSED** [review]

**Review by msasikanth — APPROVED** [review]

**Pull request merged**